base/interactiveutil.jl

```diff
@@ -274,42 +274,61 @@ function versioninfo(io::IO=STDOUT, verbose::Bool=false)
             try lsb = strip(readstring(`$(ENV["COMSPEC"]) /c ver`)) end
         end
         if !isempty(lsb)
-            println(io,     "           ", lsb)
+            println(io, "      ", lsb)
         end
         if is_unix()
-            println(io,         "  uname: ", readchomp(`uname -mprsv`))
+            println(io, "  uname: ", readchomp(`uname -mprsv`))
         end
-        println(io,         "Memory: $(Sys.total_memory()/2^30) GB ($(Sys.free_memory()/2^20) MB free)")
-        try println(io,     "Uptime: $(Sys.uptime()) sec") end
-        print(io,           "Load Avg: ")
-        print_matrix(io,    Sys.loadavg()')
-        println(io          )
-        Sys.cpu_summary(io)
-        println(io          )
     end
+
+    print(io, "  CPU: ")
+    if verbose
+        Sys.cpu_summary(io) # This should be indented somehow...
+    else
+        cpu = Sys.cpu_info()
+        println(io, cpu[1].model)
+    end
+
+    if verbose
+        println(io, "  Memory: $(Sys.total_memory()/2^30) GB ($(Sys.free_memory()/2^20) MB free)")
+        try println(io, "  Uptime: $(Sys.uptime()) sec") end
+        print(io, "  Load Avg: ")
+        print_matrix(io, Sys.loadavg()')
+        println(io)
+    end
+    println(io, "  WORD_SIZE: ", Sys.WORD_SIZE)
     if Base.libblas_name == "libopenblas" || BLAS.vendor() == :openblas || BLAS.vendor() == :openblas64
         openblas_config = BLAS.openblas_get_config()
-        println(io,         "  BLAS: libopenblas (", openblas_config, ")")
+        println(io, "  BLAS: libopenblas (", openblas_config, ")")
     else
-        println(io,         "  BLAS: ",libblas_name)
+        println(io, "  BLAS: ",libblas_name)
+    end
+    println(io, "  LAPACK: ",liblapack_name)
+    println(io, "  LIBM: ",libm_name)
+    println(io, "  LLVM: libLLVM-",libllvm_version," (", Sys.JIT, ", ", Sys.cpu_name, ")")
+
+    println(io, "Environment:")
+    for (k,v) in ENV
+        if ismatch(r"JULIA", String(k))
```

Comment:
I simply hoisted the printing of `JULIA_*` variables to always be displayed and the rest of the env vars for `verbose=true`. Add `JL_ `prefixed env vars to always show too?